packages/utils/tailwind-plugins/animations/index.js

```diff
@@ -121,6 +121,14 @@ module.exports = plugin.withOptions(
     theme: {
       extend: {
         keyframes: {
+          collapseIn: {
```

Comment:
```suggestion
          standaloneCollapseIn: {
```

see 👉 https://github.com/adevinta/spark/wiki/Conventions-for-Adding-Custom-Styles-in-Tailwind#standalone-keyframes

Reply:
As it will also be re-used by `Accordion` soon, should it still be prefixed ?

Reply:
Maybe we could name it `standaloneDisclosureIn` / `standaloneDisclosureOut`. Then, we could use that for both the collapsible and accordion elements, wdyt?

Reply:
beware, you used  `standaloneCollapseIn` / `standaloneCollapseOut`, and not `standaloneDisclosureIn` / `standaloneDisclosureOut`